frontend/src/pages/index.tsx

```
@@ -1,10 +1,9 @@
 import { useQuery } from '@tanstack/react-query'
-import { Inter } from 'next/font/google'
-import Image from 'next/image'
+import { Layout } from '@/components/Layout'
+import { Error } from '@/components/utilities/Error'
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)